elfpy/data/acquire_data.py

```diff
@@ -10,12 +10,15 @@
 from web3 import Web3
 from web3.contract.contract import Contract
 
-from elfpy.data import postgres
 from elfpy import eth, hyperdrive_interface
+from elfpy.data import postgres
 from elfpy.utils import logs as log_utils
 
 # pylint: disable=too-many-arguments
 
+# TODO fix too many branches by splitting out various things into functions
```

Comment:
can you make an issue to split up acquire_data in general? same with how I split up contract_interfaces.